modules/helm-tls/main.tf

```diff
@@ -0,0 +1,150 @@
+# ------------------------------------------------------------------------------
+# TERRAFORM / PROVIDERS CONFIG
+# ------------------------------------------------------------------------------
+
+terraform {
+  backend "gcs" {}
+}
+
+provider "tls" {}
+
+# ------------------------------------------------------------------------------
+# CA / ROOT CERTIFICATE
+# ------------------------------------------------------------------------------
+
+locals {
+  tls_dir       = "${var.custom_tls_dir == "" ? var.tiller_namespace : var.custom_tls_dir}"
+}
+
+# Root CA Key
+resource "tls_private_key" "root" {
+  algorithm   = "ECDSA"
+  ecdsa_curve = "P521"
+}
+
+# Root CA cert
+resource "tls_self_signed_cert" "root" {
+  key_algorithm   = "ECDSA"
+  private_key_pem = "${tls_private_key.root.private_key_pem}"
+
+  validity_period_hours = 26280
+  early_renewal_hours   = 8760
+
+  is_ca_certificate = true
+
+  allowed_uses = ["cert_signing"]
+
+  subject {
+    common_name  = "Helm CA"
+    organization = "Helm"
+  }
+}
+
+/*
+resource "local_file" "ca_key" {
+  filename = "${var.secrets_dir}/${local.tls_dir}/${var.helm_dir_name}/ca.key.pem"
+  content  = "${tls_private_key.root.private_key_pem}"
+}
+*/
+
+resource "local_file" "ca_cert" {
+  filename = "${var.secrets_dir}/${local.tls_dir}/${var.helm_dir_name}/ca.cert.pem"
+  content  = "${tls_self_signed_cert.root.cert_pem}"
```

Comment:
Here and all other occurences, related to above, are not consistent with the other module: `--tls-ca-cert=${local.tls_dir}/ca.cert.pem`

Reply:
Fixed.